src/protocol/RemotingCommand.h

```diff
@@ -66,6 +70,8 @@ class RemotingCommand {
   void Encode();
   static RemotingCommand* Decode(const MemoryBlock& mem);
 
+  std::string ToString();
```

Comment:
1. `TOString` or `toString`?
2. specify this method as const

Reply:
ToString() with const